Eyevinn/media-event-filter
# Feat/handle playing after ended

Tested


How to Test

Start playing, seek or play to the end. After `ended` has triggered, press or trigger `play`, or seek to anywhere in the video.

The filter should reset its internal state, causing `loaded` -> `playing` to trigger when playback starts again.

Firefox and Chrome both behave somewhat differently, in Chrome the entire action seems to be treated as a seek, while in Firefox only part of the action is treated as a seek. Regardless, media event filter will treat it as a new load and only emit `loaded`.
